staff/ana-sanjuan/mailchimp/index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Mail Chimp</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class='container-all'>
+    <header>
+        <section class='upperheader'>
+            <div class='upperheader__logo'>
+                <div class='mailchimp-Img'><img src="mailchimp/images/main_logo.png" alt=""></div>
+                <input class="upperheader__button" type="button" value='Sign Up Free'>  
+            </div>
+            <div class='upperheader__nav-container'>
+                <p class='upperheader__p'><a href="index.html"> Log In</p></a>
+                <nav> 
+                    <ul class='upperheader__nav__ulist'>
```

Comment:
Hey Ana, "[According to BEM method, block structure should be flattened; you do not need to reflect nested DOM structure of the block](
http://getbem.com/faq/#css-nested-elements)" . It is recommended to avoid using a grandchild.